scripts/index-demo-page.js

```diff
@@ -18,9 +18,10 @@
     var id = selectedOption.value;
 
     window.vhs.representations().forEach(function(rep) {
-      rep.enabled(rep.id === id);
+      rep.playlist.disabled = rep.id !== id;
```

Comment:
The original implementation changes playlists every time enabled is changed. Now we only change once when we call `smoothQualityChange_` below this. Since all playlists other than the one we want are disabled, the one we want is selected and changed to.